manifests/profile/www_lib/vhosts/deepblue.pp

```diff
@@ -74,6 +74,11 @@
         rewrite_cond => '%{REQUEST_URI} !^((\/?|/index.html)$|/splash/|/cosign/valid)',
         rewrite_rule => '^(.*)$	http://bulleit-2.umdl.umich.edu:8080$1 [P]'
       },
+      {
+        comment      => 'Deep Blue Preservation redirect',
+        rewrite_cond => '^(.*)$ https://deepblue.lib.umich.edu/static/about/deepbluepreservation.html [P]',
+        rewrite_rule => '^(.*)$	https://www.lib.umich.edu/about-us/policies/digital-repository-services-digital-preservation-policy/registered-formats-and [P]'
```

Comment:
The flags should be `[R=permanent,L]`, which makes it a (permanent) 301 [R]edirect and makes this the [L]ast rule in the evaluation -- immediate client redirect.

The pattern won't need any matching groups since it's a literal redirect. It should have the head anchor, though. That is, I think `^/static/about/deepbluepreservation.html` is about the best pattern.

Reply:
I think I have this corrected now.  cleaned up that spacing gap too. (probably a tab)